src/generators/app-lit-element-ts/index.js

```diff
@@ -51,8 +51,8 @@ export const TsAppLitElementMixin = subclass =>
 
       if (this.options.scaffoldFilesFor && this.options.scaffoldFilesFor.includes('demoing')) {
         this.copyTemplate(
-          `${__dirname}/templates/_my-app.stories.ts`,
-          this.destinationPath(`./stories/${tagName}.stories.ts`),
+          `${__dirname}/templates/_my-app.stories.js`,
+          this.destinationPath(`./stories/${tagName}.stories.js`),
```

Comment:
Was there an issue with the stories being run as TS?

Reply:
It could be done, there are some pros and cons to both so I opted for the simpler option since the story files usually aren't that big.

Reply:
Are there any specific cons you can share re: using TS here? I feel a bit strange using JS in the TS generator _sometimes_ without clear cause. "Clear" being less powerful that "good" but more powerful that "no" in the order of cause preference...

Reply:
The downside is mostly that the stories appear in the `dist` folder, and thus get published along with the other files. But this is a problem with the testing configuration too, so you're right it's better to be consistent.

Reply:
Made the stories typescript now